test/Serilog.Sinks.OpenTelemetry.Tests/Serilog.Sinks.OpenTelemetry.Tests.csproj

```diff
@@ -15,6 +16,7 @@
     <PackageReference Include="xunit" Version="2.4.2" />
     <PackageReference Include="PublicApiGenerator" Version="11.0.0" />
     <PackageReference Include="Shouldly" Version="4.2.1" />
+    <PackageReference Include="OpenTelemetry.SemanticConventions" Version="1.0.0-rc9.9" />
```

Comment:
Keeping this in the tests means we'll be more likely to spot any drift/typos.

Reply:
To be honest, I am not sure if it will be released in the same way, but it can be easily fixed after release 1.0.